### ISSUE TITLE
Missing AnalyticsResponse on sessions first hits

### ISSUE THREAD

**Issue by hannoguenther** [adobe/aepsdk-analytics-android#58]
Hi, we currently have warnings with missing AnalyticsResponses for our sessions first "Track State" Hits in Adobe Assurance.
While debugging into Adobe Analytics source code, I found that `timestamp` was equal to ` analyticsState.lastResetIdentitiesTimestampSec`.
Which happens quite a lot on early hits, as timestamps are recorded in seconds and not in milliseconds.

Problem: No AnalyticsResponse is send due to `timestamp > analyticsState.lastResetIdentitiesTimestampSec`.

Proposal: `timestamp >= analyticsState.lastResetIdentitiesTimestampSec`

See https://github.com/adobe/aepsdk-analytics-android/blob/066c980e58d673dca2e2344aee1c201172fe8f19/code/analytics/src/main/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsHitProcessor.kt#L182

**Comment by cacheung** [comment]
@hannoguenther for the report, we are looking into.

**Comment by cacheung** [comment]
@hannoguenther Can you provide more info about your implementation?  Do you call resetIdentities on each app launch? Or where do you call resetIdentities? Can you provide us an Assurance session id which demonstrates the scenario for debugging?

**Comment by hannoguenther** [comment]
SessionID: 14fe181c-39af-497c-b433-c39efb3e7feb

Hi @cacheung, I don't call resetIdentities.
It's just a normal setup with an early `MobileCore.trackState("Loading Screen (App Start)", ...)` call.

```
MobileCore.setApplication(application)
MobileCore.configureWithAppID(application.getString(R.string.adobe_app_id))
MobileCore.setLogLevel(LoggingMode.VERBOSE)
MobileCore.registerExtensions(
    listOf(
        Analytics.EXTENSION,
        Identity.EXTENSION,
        Lifecycle.EXTENSION,
        Signal.EXTENSION,
        Assurance.EXTENSION,
    )
) { Log.i("AdobeSetup","AEP Mobile SDK is initialized") }
application.registerActivityLifecycleCallbacks(object : LifecycleCallbacksBase() {
    override fun onActivityResumed(activity: Activity) {
        MobileCore.lifecycleStart(null)
    }

    override fun onActivityPaused(activity: Activity) {
        MobileCore.lifecyclePause()
    }
})
```

<img width="1737" alt="image" src="https://user-images.githubusercontent.com/37990037/225248142-57de6730-1556-45dc-b24a-dff18a600de8.png">

**Comment by cacheung** [comment]
@hannoguenther thanks for the info.  We are looking into this.

**Comment by emdobrin** [comment]
@hannoguenther thank you for reporting this issue and providing us with the necessary information to debug further. We identified the root cause of the issue and implemented a fix, we are planning to release this fix to production in the week of March 27th. 

The issue should not affect the delivery of these track requests that are sent early from your app to Adobe Analytics, so you should be able to see the data in your report suite, but you will not be able to see the post-processed data in Adobe Assurance for these hits for now.

I hope this helps, also feel free to check the fix and test with the dev-v2.0.2 branch in the meantime.

**Comment by hannoguenther** [comment]
@emdobrin Thanks very much! I tried out dev-v2.0.2 branch and it works perfect!
